test/StringWrapper/IconvTest.php

```diff
@@ -24,6 +28,19 @@ protected function setUp(): void
             }
         }
 
+        /**
+         * ext-iconv is not properly supported on Alpine Linux, hence, we skip the tests for now
+         *
+         * @see https://github.com/nunomaduro/phpinsights/issues/43
+         * @see https://github.com/docker-library/php/issues/240#issuecomment-353678474
+         */
+        if (file_exists('/etc/os-release') && is_readable('/etc/os-release')) {
```

Comment:
Can you try with `php_uname` function ?

Reply:
When running locally a Docker container with Alpine Linux, `php_uname`  returns the following:

```
Linux 7ffea1b1e4ef 5.10.60.1-microsoft-standard-WSL2 #1 SMP Wed Aug 25 23:20:18 UTC 2021 x86_64
```

That is the Kernel of the host system (WSL2 on Windows 10). So I dont' think using `php_uname` will work here.

Reply:
:ok_hand: